.golangci.yaml

```diff
@@ -31,7 +33,7 @@ linters-settings:
         # Default: true
         skipRecvDeref: false
 
-  gomnd:
+  mnd:
```

Comment:
why the rename?

Reply:
I don't know why, but golangci replaced it. It still supports gomnd as an alias but reports warnings, so I went ahead and renamed it.

https://golangci-lint.run/product/changelog/#v1580